<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/logoIcon.png</normaloff>:/resources/logoIcon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="banner">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/bannerApp.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="formFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <widget class="QLabel" name="loginUsernameLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>198</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 7px;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginUsernameInput">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
padding-left: 8px;
background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="loginPasswordLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>258</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 7px;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginPasswordInput">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
      padding-left: 8px;
      background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #E1E1E1;
border-radius: 8;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
border-width: 1px;
border-color: #ADADAD;
border-style: solid;
}

QPushButton:hover {
widget-animation-duration: 100;
background-color: #E4EFF9;
border-color: #0C7BD4;
}

QPushButton:pressed {
background-color: #CDE4F7;
border-color: #00559B;
}</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginErrorLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loginText">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>436</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#3C505B</string>
    </property>
    <property name="text">
     <string>Register a new account ―&gt;</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="registerText">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>436</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>11</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#3C505B</string>
    </property>
    <property name="text">
     <string>&lt;― Back to login page</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="registerEmailLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>173</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 7px;</string>
    </property>
    <property name="text">
     <string>Email address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerEmailInput">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>195</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
      padding-left: 8px;
      background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="registerUsernameLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>228</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 7px;</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerUsernameInput">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
      padding-left: 8px;
      background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="registerPasswordLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>283</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 7px;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="registerPasswordInput">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>305</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 8px;
padding-left: 8px;
background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #E1E1E1;
border-radius: 8;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
border-width: 1px;
border-color: #ADADAD;
border-style: solid;
}

QPushButton:hover {
widget-animation-duration: 100;
background-color: #E4EFF9;
border-color: #0C7BD4;
}

QPushButton:pressed {
background-color: #CDE4F7;
border-color: #00559B;
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QLabel" name="registerErrorLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loggedNewButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>181</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	text-align: left;
}
QPushButton:hover {
	text-align: left;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string> New Document</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/newDoc.png</normaloff>:/resources/newDoc.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loggedOpenButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>191</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	text-align: left;
}
QPushButton:hover {
	text-align: left;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string> Open Document</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/openDoc.png</normaloff>:/resources/openDoc.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loggedLogoutButton">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>400</y>
      <width>111</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	text-align: left;
}
QPushButton:hover {
	text-align: left;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/logOut.png</normaloff>:/resources/logOut.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="loggedSettingsButton">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>190</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #707070;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/ic_settings_18pt_2x.png</normaloff>:/resources/ic_settings_18pt_2x.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="popupMode">
     <enum>QToolButton::InstantPopup</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loggedWelcomeLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="loggedAvatar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/resources/avatar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="loggedUsernameLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>dario.patti90</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loggedURIButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>186</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	text-align: left;
}
QPushButton:hover {
	text-align: left;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string> Open From URI</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/openURI.png</normaloff>:/resources/openURI.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="informationWhiteFrame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #F2ECFF;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="informationLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>240</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string></string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="informationButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
       background-color: #E1E1E1;
       border-radius: 8;
       padding: 3px;
       padding-left: 5px;
       padding-right: 5px;
       border-width: 1px;
       border-color: #ADADAD;
       border-style: solid;
       }

       QPushButton:hover {
       widget-animation-duration: 100;
       background-color: #E4EFF9;
       border-color: #0C7BD4;
       }

       QPushButton:pressed {
       background-color: #CDE4F7;
       border-color: #00559B;
       }</string>
     </property>
     <property name="text">
      <string></string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="informationImage">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>loginUsernameLabel</zorder>
   <zorder>loginUsernameInput</zorder>
   <zorder>loginPasswordInput</zorder>
   <zorder>loginPasswordLabel</zorder>
   <zorder>loginButton</zorder>
   <zorder>loginErrorLabel</zorder>
   <zorder>loginText</zorder>
   <zorder>registerText</zorder>
   <zorder>registerEmailLabel</zorder>
   <zorder>registerEmailInput</zorder>
   <zorder>registerUsernameLabel</zorder>
   <zorder>registerUsernameInput</zorder>
   <zorder>registerPasswordLabel</zorder>
   <zorder>registerPasswordInput</zorder>
   <zorder>registerButton</zorder>
   <zorder>registerErrorLabel</zorder>
   <zorder>informationWhiteFrame</zorder>
   <zorder>loggedNewButton</zorder>
   <zorder>loggedOpenButton</zorder>
   <zorder>loggedLogoutButton</zorder>
   <zorder>loggedSettingsButton</zorder>
   <zorder>loggedWelcomeLabel</zorder>
   <zorder>loggedAvatar</zorder>
   <zorder>loggedUsernameLabel</zorder>
   <zorder>loggedURIButton</zorder>
  </widget>
  <widget class="QFrame" name="titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>5</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/closeButton.png</normaloff>:/resources/closeButton.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>5</y>
      <width>21</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/resources/minimizeButton.png</normaloff>:/resources/minimizeButton.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="formLogo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>45</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/resources/logoApp.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="verticalSeparator">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>-1</y>
     <width>1</width>
     <height>482</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <action name="actionChangeUsername">
   <property name="text">
    <string>Change your username</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>Change your password</string>
   </property>
  </action>
  <action name="actionChangeAvatar">
   <property name="text">
    <string>Change your avatar</string>
   </property>
  </action>
  <zorder>formFrame</zorder>
  <zorder>formLogo</zorder>
  <zorder>titleBar</zorder>
  <zorder>banner</zorder>
  <zorder>verticalSeparator</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
